### ISSUE TITLE
Categorical abundance

### ISSUE THREAD

**Issue by timcdlucas** [zoonproject/zoon#310]
Currently we have count abundance. Abundance categories requested.

**Comment by AugustT** [comment]
Does this mean 'low', 'medium', 'high'? Are there methods that use this kind of data? the format would been to be defined, eg 'low', 'medium', 'high', but much better would be numeric representation (1:3)

**Comment by timcdlucas** [comment]
Yeah that was the idea. According to @ali-johnston there are methods that are being used.

I think a factor would be sensible. So yes, 1:3 with labels and clear that 2 - 1 != 3 - 2 etc.

**Comment by goldingn** [comment]
Apparently ordered factors are supported in R: http://www.dummies.com/how-to/content/how-to-work-with-ordered-factors-in-r.html
